packages/core/src/components/tree-view/tree-view.tsx

```diff
@@ -104,7 +107,7 @@ export class TreeView implements ComponentInterface {
       }
 
       TreeView.setFocus(tree, contentElement);
-      this.clickItem.emit([...ancestors, item]);
+      this.clickItem.emit({ path: [...ancestors, item], originalEvent: event as PointerEvent });
```

Comment:
Kan in de signature op regel 96 de typering niet al goed worden gezet?

Reply:
Bij nader inzien is het helemaal geen `PointerEvent` maar een `MouseEvent`.